### ISSUE TITLE
Warn users about missing Command line tools 14.1 on launch

### ISSUE THREAD

**Issue by mondoo-Alexander** [mondoohq/cnquery#420]
**What is not working as you expected it?**
Currently, it is not possible to install and/or update cnspec / cnquery through brew / the cli in general on the Mac if users already upgraded to macOS Ventura, because the required command line tools 14.1 have not been released officially.

**Where on the platform does it happen?**
Terminal on macOS Ventura when trying to install or update cnquery/cnspec 

**How do we replicate the issue?**
Run brew install cnspec/cnquery or brew upgrade cnspec/cnquery on any Mac on macOS Ventura

**Suggestion for a solution**
We can currently not do anything about the missing developer tools but it is possible to still install cnspec and cnquery with the binaries, so in order to make the launch experience smoother and to not leave users cornered with nothing to do, we could at least put out a tweet or a blog post what to do if you have already adopted macOS Ventura. Especially because the solution brew suggests doesn't work and creates a loop.

**Comment by chris-rock** [comment]
https://rakhesh.com/mac/xcode-14-and-command-line-tools-on-macos-ventura/ seems to solve the problem for homebrew.

**Comment by mondoo-Alexander** [comment]
Tried it and it fixes this, but the usual macOS UX would be to remove the command line tools and run xcode-select --install, or to open settings and see if updates are available, which currently yields no results. So unless people google it, they won't find it, because brew instructions (same as what I described above) will result in installing the old v13.x again, trapping the user in a loop.

It's certainly an Apple problem and 100% their fault but maybe we could still simply tweet out that:
"macOS Ventura early adopters can also try cnspec and cnquery by simply downloading the release candidate of the XCode Command Line tools from https://developer.apple.com/download/all/"

**Comment by tas50** [comment]
The brew instructions are not the best here, but I'm not sure how much of that we should be working around.

**Comment by mondoo-Alexander** [comment]
Agreed, we shouldn't work around anything, I am merely suggesting sending out a tweet

**Comment by charlesjohnson** [comment]
XCode CLI Tools 14.1 just shipped this morning, might render this moot?
